department-of-veterans-affairs/abd-vro
# add temp cron to run secrel action

## What was the problem?
We want the secrel action to run daily so we can monitor the results without manually triggering the event.

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/abd-vro/issues/3031

## How does this fix it?[^1]
This commit is a test to verify cron functionality in github actions.  The scan is set currently to run every 5 minutes; when we validate the scan is running on a timer the cron will be changed to run daily at 1100 UTC/0700 ET.

## How to test this PR
- Step 1 examine the secrel runs on the actions page and verify it is running on a schedule


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
